### ISSUE TITLE
Feature - Blackout Mode

### ISSUE THREAD

**Issue by quittle** [quittle/a11y-ally#18]
Black out all elements with no accessibility text. It would call out better than highlighting errors the inability to reason about an app if you can't see what a button is
